packages/next/src/lib/metadata/generate/basic.tsx

```diff
@@ -64,7 +70,15 @@ export function BasicMeta({ metadata }: { metadata: ResolvedMetadata }) {
       <link
         rel="manifest"
         href={metadata.manifest.toString()}
-        crossOrigin="use-credentials"
+        // If it's same origin, and it's a preview deployment,
+        // including credentials for manifest request.
+        crossOrigin={
+          metadataBase &&
+          metadataBase.origin === manifestOrigin &&
+          process.env.VERCEL_ENV === 'preview'
```

Comment:
Is this right?

Why isn't it anytime it's a preview if the manifestOrigin is not set we infer it is to same origin and the credentials rule applies